guardian/manage-frontend
# Use Help Centre header as header for entire site

## What does this change?

Reuse the same header that uses the full 'The Guardian' logo across HelpCentre and Manage My Account. Slightly adjust it to have a smaller top margin as per product switching designs. 



## How to test

Visit MMA while signed in - see The Guardian logo on the right and My Account dropdown on the left. Visit `/help-centre` and see the same header. Sign out while on the help centre and see 'Sign In' on the left of the header.

Acceptance Criteria
- Roundel to be replaced with The Guardian logo
- The Guardian logo to hyperlink to The Guardian website
- 'My account' dropdown to be shifted to the left side on desktop, replacing the 'My Account' link, with dropdown functionality remaining the same as it currently is.



## How can we measure success?



## Have we considered potential risks?



## Images

<img width="1355" alt="image" src="https://user-images.githubusercontent.com/114918544/218767503-8624e423-edda-4e65-b7fb-74e3a985c287.png"><img width="513" alt="image" src="https://user-images.githubusercontent.com/114918544/218767429-f580a355-626e-48cd-b723-294c98a2e6f9.png">



## Accessibility
